fdbserver/Ratekeeper.actor.cpp

```diff
@@ -764,6 +764,7 @@ ACTOR Future<Void> monitorServerListChange(
 			tr = Transaction(self->db);
 			wait(delay(SERVER_KNOBS->SERVER_LIST_DELAY));
 		} catch (Error& e) {
+			TraceEvent("RatekeeperGetSSListError", self->id).suppressFor(1.0).error(e);
```

Comment:
I don't think this is enough info to tell:
Is it due to RK role is too slow (say due to slow machine) or due to read-request is slow. 

We need to log the time before L741 `			vector<std::pair<StorageServerInterface, ProcessClass>> results = wait(getServerListAndProcessClasses(&tr));
`. 
If RK's machine is slow, it will wait for longer than SERVER_LIST_DELAY time in each loop.

Reply:
You mean you just want to see if ratekeeper is saturated so that it can't run this query? That sounds like something that would probably be better solved more generally.

Reply:
For instance, `NetworkMetrics` will have details about how busy this process is and to what degree it is starved at various priorities.

Reply:
Added.